lms/djangoapps/teams/views.py

```diff
@@ -584,9 +584,21 @@ def delete(self, request, team_id):
         """DELETE /api/team/v0/teams/{team_id}"""
         team = get_object_or_404(CourseTeam, team_id=team_id)
         self.check_object_permissions(request, team)
+        memberships = list(CourseTeamMembership.get_memberships(team_id=team_id))
```

Comment:
Minor nit: the explicit call to `list` isn't really necessary; that happens implicitly when looping over the queryset on line 595.